### PR TITLE
Removing errors while receiving an empty response

### PR DESCRIPTION
**Pull request by victorperin**
Removing showing errors like: "error:   Parsing error:"
When an empty response is received.

This is an annoying error when you are streaming an tag that's not all time updated or you're seeing your timeline.

### CODE REVIEW

File: src/common/api.js [hunk 1]
```diff
@@ -103,7 +104,8 @@ exports.timeline = function (user, options, callback) {
         try {
             return JSON.parse(str);
         } catch(e) {
-            cli.log.error('Parsing error:' + str + '\n');
+            if(str!="")
```
Comment (voronianski) [inline src/common/api.js]: @victorperin please make the code style similar to what we have in repo, also add strict comparison and single quotes:

```javascript
if (str !== '') {
   cli.log.error('Parsing error:' + str + '\n');
}
```

**Comment by victorperin** [comment]
Alright, changing...

**Comment by victorperin** [comment]
Oh, I'm sorry. Did you plan to implement something to test it on travis?
Like it: https://github.com/dsheiko/jscodesniffer ?

**Comment by voronianski** [comment]
@victorperin I have `.eslintrc` inside the repo so I advice you to have [ESLint](http://eslint.org/) for your IDE.

**Pull request merged**